src/utils/jwt.ts

```diff
@@ -14,10 +14,12 @@ import { errorsToError } from "./conversions";
 
 const ExpireJWT = t.exact(
   t.interface({
-    exp: t.number,
+    exp: t.number
   })
 );
 
+type ExpireJWT = t.TypeOf<typeof ExpireJWT>;
```

Comment:
Since the object is already defined, I declared the type just to use it in the `extractJwtRemainingValidTime` function